public/app/features/alerting/unified/RuleList.test.tsx

```diff
@@ -88,7 +89,12 @@ const mocks = {
 const renderRuleList = () => {
   locationService.push('/');
 
-  return render(<RuleList />);
+  return render(
+    <TestProvider>
```

Comment:
Wonder why this `TestProvider` is needed here (and makes the tests pass) since it has the same providers but in different order? Wonder if we need a custom wrappers here since the idea is to get rid of the `TestProvider` eventually?